test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/TagHelperOutputExtensionsTest.cs

```diff
@@ -979,8 +979,8 @@ public bool Equals(TagHelperAttribute attributeX, TagHelperAttribute attributeY)
                 // Normal comparer (TagHelperAttribute.Equals()) doesn't care about the Name case, in tests we do.
```

Comment:
Please remind me why this private class exists. Don't we have the same thing in Razor?